cmd/zoekt-mirror-github/main.go

```diff
@@ -311,7 +306,9 @@ func cloneRepos(destDir string, repos []*github.Repository) error {
 			"zoekt.github-subscribers": itoa(r.SubscribersCount),
 			"zoekt.github-forks":       itoa(r.ForksCount),
 
-			"zoekt.archived": marshalBool(archived),
+			"zoekt.archived": marshalBool(r.Archived != nil && *r.Archived),
+			"zoekt.fork":     marshalBool(r.Fork != nil && *r.Fork),
+			"zoekt.public":   marshalBool(r.Private != nil && !*r.Private),
```

Comment:
Hmm, maybe this should be
```suggestion
			"zoekt.public":   marshalBool(r.Private == nil || !*r.Private),
```
I'm not sure why all of these are pointers in the first place.

This is what I get for not actually testing these commands, I will do that tomorrow.

Reply:
In my testing, I couldn't find a situation in which these pointers could be nil, but I'll make the change so that it defaults to public, which I think is more conceptually appropriate.